lib/Parser/Parse.cpp

```diff
@@ -4757,16 +4783,23 @@ ParseNodePtr Parser::ParseMemberList(LPCOLESTR pNameHint, uint32* pNameHintLengt
 
         if (buildAST)
         {
-            Assert(pnodeArg->pnode2 != nullptr);
-            if (pnodeArg->pnode2->nop == knopFncDecl)
+            if (useSpread)
             {
-                Assert(fullNameHintLength >= shortNameOffset);
-                ParseNodeFnc * pnodeFunc = pnodeArg->pnode2->AsParseNodeFnc();
-                pnodeFunc->hint = pFullNameHint;
-                pnodeFunc->hintLength = fullNameHintLength;
-                pnodeFunc->hintOffset = shortNameOffset;
+                AddToNodeListEscapedUse(&pnodeList, &lastNodeRef, pnodeSpread);
```

Comment:
Possible for pnodeSpread to be nullptr still here?

Reply:
Wait, so does that mean:
```js
let o = { a: 812, b: 1208, ...(c) };
```

Would cause a nullptr dereference?